packages/flame/test/components/sprite_animation_component_test.dart

```diff
@@ -138,5 +138,41 @@ void main() async {
       game.update(0.1);
       expect(game.components.length, 1);
     });
+
+    test('Last animation frame is not skipped', () {
+      // See issue #895
```

Comment:
We could have the link for the issue here instead of only the number

Reply:
done